### DISCUSSION TITLE
Approval problems

### DISCUSSION THREAD

**Discussion by Hastes** [WalletConnect/walletconnect-monorepo#4251] [Developer Support]
I got that message in dashboard, but we don't have any labels "Wallet Connect" in extension, only in console logs.

![telegram-cloud-photo-size-2-5377651053387961693-y](https://github.com/WalletConnect/walletconnect-monorepo/assets/19662992/a63bfc40-c6a4-4244-8739-208ecef01fb9)

Can I get more info?

**Comment by glitch-txs** [comment]
hi, what's your project id?

**Reply by Hastes** [reply]
Hi
1ee56a25a2dad471b92feb59898b7aa6

**Reply by glitch-txs** [reply]
You set the EIP-6963 RDNS field but this is only for in-app browsers or browser extensions that support the EIP

**Reply by Hastes** [reply]
Our wallet also has a browser extension (https://chromewebstore.google.com/detail/dextrade/nmladckinifchdidkoaagbcpnieocoah?hl=th) that supports EIP-6963.

Our new build with EIP support is currently awaiting approval in chromewebstore. Should I request approval from WalletConnect again?

**Reply by glitch-txs** [reply]
Yes please, could you also send an email to cloud@walletconnect.com with these details
